tools/util_linux.go

```diff
@@ -22,6 +22,10 @@ const (
 	BtrfsIocClone = C.BTRFS_IOC_CLONE
 )
 
+func CloneFileSupported() bool {
```

Comment:
Good idea.
Splatted as https://github.com/git-lfs/git-lfs/pull/3754.

Reply:
I think it would actually be nice to include it in this pull request. If you want to provide a distinction between the code, it's fine to rebase in a separate commit, but it's not necessary.